.go-version

```diff
@@ -0,0 +1 @@
+1.19.1
```

Comment:
I'm moving all my projects to use go.mod for version info. So this file will be ignored and confusing. Would you mind removing it?

Reply:
Will consul-template not be switching to the common release tooling?  The CRT actions use the .go-version file to determine what go version to install.

Reply:
Consul-template already uses CRT. CRT provides a generic example config for the github action workflows that needs to be tweaked and maintained for each project. It is not a fixed script. I didn't want/need the detailed control of .go-version but I did need it to use the appropriate compiler version for the code (as is specified in go.mod). Github actions supports doing that and it was only a [minor change to the build.yml](https://github.com/hashicorp/consul-template/commit/47c9feceb89932789301ae679ee89de61b7011a5).